og.routing.yml

```diff
@@ -1,5 +1,38 @@
 # Routes for Organic groups.
 
+# OgRole entity routes.
```

Comment:
Added section for the OgRole entity to avoid merge conflicts when merging routes for other entities.